content/sensu-go/6.3/operations/business-service-monitoring-sdk.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Business service monitoring SDK"
+linkTitle: "Business Service Monitoring SDK"
+description: "Based on JavaScript expressions, Sensu's business service monitoring SDK provides additional functionality for Sensu rule templates that evaluate components. Read the reference doc to learn about the business service monitoring SDK."
+weight: 70
+version: "6.3"
+product: "Sensu Go"
+platformContent: false 
+menu:
+  sensu-go-6.3:
+    parent: operations
+---
+
+Sensu's business service monitoring (BSM) feature uses a dedicated SDK of JavaScript-based expressions that provide additional functionality.
+
+BSM SDK expressions are defined in [rule templates][3], so they act in the context of determining whether aggregate data derived from a component’s selection of Sensu Go events should trigger a rule-based event.
+BSM SDK expressions always receive a single event and some information about that event, like `event.timestamp` or `event.check.interval`.
+
+BSM SDK expressions always return either `true` or `false`.
+They are evaluated by the [Otto JavaScript VM][1] as JavaScript programs.
```

Comment:
These last three sentences are guesses based on the SQE reference (https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-filter/sensu-query-expressions/)